### ISSUE TITLE
object 'validator_ml_linear_regression' of mode 'function' was not found

### ISSUE THREAD

**Issue by zj56** [sparklyr/sparklyr#3273]
R version 4.2.1 (2022-06-23 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 25145)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Simplified)_China.utf8  LC_CTYPE=Chinese (Simplified)_China.utf8   
[3] LC_MONETARY=Chinese (Simplified)_China.utf8 LC_NUMERIC=C                               
[5] LC_TIME=Chinese (Simplified)_China.utf8    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] MASS_7.3-57    stringr_1.4.0  tibble_3.1.7   magrittr_2.0.3 ggplot2_3.3.6  dplyr_1.0.9   
[7] sparklyr_1.7.7

```r
cv <- ml_cross_validator(
  sc,
  estimator = pipeline,
  estimator_param_maps = list(
    standard_scaler = list(with_mean = c(TRUE, FALSE)),
    linear_regression = list(
      elastic_net_param = c(0.25, 0.75),
      reg_param = c(0.1, 0.05)
    )
  ),
  evaluator = ml_regression_evaluator(sc, label_col = "medv"),
  num_folds = 10, parallelism = 4, seed=1111)

# 
Error in get(paste0("validator_", ml_map_class(cl)), envir = asNamespace(ml_map_package(cl)),  :   object 'validator_ml_linear_regression' of mode 'function' was not found
[code.zip](https://github.com/sparklyr/sparklyr/files/9009910/code.zip)

**Comment by zj56** [comment]
you can download it from 
[code.zip](https://github.com/sparklyr/sparklyr/files/9009910/code.zip)

**Comment by edgararuiz** [comment]
Reprex: 

```r
library(sparklyr)
#> 
#> Attaching package: 'sparklyr'
#> The following object is masked from 'package:stats':
#> 
#>     filter
sc <- spark_connect(master = "local")
mtcars_tbl <- copy_to(sc, mtcars)

pipeline <- ml_pipeline(sc) %>%
  ft_r_formula(mpg ~ .) %>%
  ml_linear_regression()

cv <- ml_cross_validator(
  sc,
  estimator = pipeline,
  estimator_param_maps = list(
    linear_regression = list(
      elastic_net_param = c(0.25, 0.75),
      reg_param = c(0.1, 0.05)
    )
  ),
  evaluator = ml_regression_evaluator(sc, label_col = "medv"),
  num_folds = 10, 
  parallelism = 4, 
  seed=1111
  )
#> Error in get(paste0("validator_", ml_map_class(cl)), envir = asNamespace(ml_map_package(cl)), : object 'validator_ml_linear_regression' of mode 'function' was not found

spark_disconnect(sc)
```

**Comment by edgararuiz** [comment]
Working on fix on this branch: https://github.com/sparklyr/sparklyr/tree/bugfix-3273

**Comment by edgararuiz** [comment]
Fixed by #3279

**Comment by zj56** [comment]
Thank you very much!!!
